SQL/2016-03-25-GenomicBrowser-CPG.sql

```diff
@@ -0,0 +1,64 @@
+-- ALTER TABLE genome_loc ADD Description text, MODIFY Strand enum ('forward', 'reverse');
```

Comment:
Patch should also add unique key and index to genome_loc table, to match schema change